### PR TITLE
Fixed notice when match fails.

### PR DESCRIPTION
**Pull request by Daniel-KM**
Hi,
I have a pdf with "PageLabelBegin" and the match failed. So for all unmanaged metadata, there should be a check.
Sincerely,

### CODE REVIEW

File: src/InfoFields.php [hunk 1]
```diff
@@ -87,7 +86,11 @@ private function parseData($dataString)
                 continue;
             }
 
-            preg_match('/([^:]*): ?(.*)/', $trimmedLine, $match);
+            $match = array();
+            $result = preg_match('/([^:]*): ?(.*)/', $trimmedLine, $match);
```
Comment (mikehaertl) [inline src/InfoFields.php]: A matter of style, I know: But I'd say a simple `if (!preg_match(...)) {` is enough here.

**Comment by mikehaertl** [comment]
Could you maybe add the problematic output you have to the example in the comment block on top of this function? Maybe we could also have a comment about what is filtered out.

**Comment by mikehaertl** [comment]
@Daniel-KM If you don't find time I can also add the requested changes. But maybe you could paste the original problematic output of pdftk here. I'd like to have a realistic example in the comment block.

**Comment by mikehaertl** [comment]
@Daniel-KM Thanks for the update. I assume the output you try to fix looks something like this:

```
PageLabelBegin
PageLabelNewIndex: 1
PageLabelStart: 1
PageLabelPrefix: cover
PageLabelNumStyle: NoNumber
PageLabelBegin
PageLabelNewIndex: 2
PageLabelStart: 1
PageLabelPrefix:  IFC-
PageLabelNumStyle: DecimalArabicNumerals
```

I've done some research and think that there's a better way to fix this. We already parse similar blocks that begin with `InfoBegin`, `BookmarkBegin` and `PageMediaBegin`. I think we should extend that and also parse `PageLabelBegin` similar to what you see herehttps://github.com/mikehaertl/php-pdftk/blob/master/tests/InfoFieldsTest.php#L52.

If you confirm that the above output is what causes you trouble I will implement a fix.

**Comment by Daniel-KM** [comment]
Yes, 
```
PageLabelBegin
PageLabelNewIndex: 01
PageLabelStart: 1
PageLabelNumStyle: DecimalArabicNumerals
```
Even if you include this metadata, there should be a check for preg_match.

**Comment by mikehaertl** [comment]
> Even if you include this metadata, there should be a check for preg_match.

Sure, I'm already working on an improved fix.

**Comment by mikehaertl** [comment]
@Daniel-KM Closing this in favour of https://github.com/mikehaertl/php-pdftk/pull/171. Would be nice if you could have a look

Thanks for bringing up this issue!

**Pull request closed without merging**